versioned_docs/version-8.2/components/operate/userguide/resolve-incidents-update-variables.md

```diff
@@ -119,8 +119,6 @@ We’ll publish a message that will be correlated with the instance, so we can a
 
 In the Operate interface, you should now see the process instance has an <!-- FIXME: [“Incident”](/reference/incidents.html) --> incident, which means there’s a problem with process execution that must be fixed before the process instance can progress to the next step.
```

Comment:
LOL what is this?! 😆 Can you follow up with the Operate team on a separate PR?

Reply:
Lol yes. Will launch a separate issue 👍